src/components/Card.jsx

```diff
@@ -1,13 +1,20 @@
 import styled from "styled-components";
 import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
 
 const CardWrapper = styled.section`
   display: flex;
   flex-direction: column;
   justify-content: space-evenly;
+  align-items: center;
 
   li {
     list-style: none;
```

Comment:
If you add `list-style: none` to your `li` element then you should add as attribute `role="list"` to your `ul` element in line 38. This is for accessibility reasons as a screenreader can't identify a list when `list-style` ist set to `none`. But if you add the role attribute to your list it is fine. 👍

Reply:
Hey! Thank you very much for your comment. I will definitely add those attributes right after the merge directly in the main branch. I hope that's fine? :)